schemas/event.yaml

```diff
@@ -0,0 +1,146 @@
+$schema: http://json-schema.org/draft-07/schema#
```

Comment:
I'm a little confused (and I may just be out of the loop here), but why aren't we using an OpenAPI definition for this?

Reply:
@awood the idea was to define a JSON schema for communication from external systems to Swatch via a Kafka topic. I'm not opposed to putting it in the API spec (though it feels like we should define an (at least stubbed) API endpoint for it if we do). Thoughts?

Reply:
fwiw you'll eventually be able to enforce schemas on kafka topic via kafka itself. I don't think there is any harm in enforcing it in 2 places though.

Reply:
![4ql3zv](https://user-images.githubusercontent.com/19955562/102365515-a36d9480-3f85-11eb-9f6d-4602e5f8f8c0.jpg)